source/connect.txt

```diff
@@ -9,5 +9,147 @@ Connect to a Deployment
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+
+Ensure that a MongoDB deployment is running before attempting to 
+connect to it with ``mongosh``. The connection method you use depends 
+on your deployment type.
+
+Local MongoDB Instance on Default Port
+--------------------------------------
+
+You can run ``mongosh`` without any command-line options
+to connect to a MongoDB instance running on your **localhost** with 
+**default port** 27017:
+
+.. code-block:: sh
+
+   mongosh
+
+Local MongoDB Instance on a Non-Default Port
+--------------------------------------------
+
+To specify a port to connect to on localhost, use the 
+:manual:`connection string </reference/connection-string>` format. For 
+example, to connect to a MongoDB instance running on localhost with a 
+non-default port 28015:
+
+.. code-block:: sh
+
+   mongosh "localhost:28015"
+
+MongoDB Instance on a Remote Host
+---------------------------------
+
+To specify a remote host and port, use a :manual:`connection string
+</reference/connection-string>`. For example, to connect to a MongoDB
+instance running on a remote host machine on port 28015:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://mongodb0.example.com:28015"
+
+  .. note:: Connecting to |service|
+
+     If your remote host is a |service-fullname| cluster, you can copy 
+     your connection string from the |service| UI. To learn more, see 
+     :atlas:`Connect to a Cluster <connect-to-cluster/#use-the-connect-dialog-to-connect-to-your-cluster>`.
+
+Connection Considerations
+-------------------------
+
+Authentication
+~~~~~~~~~~~~~~
+
+To connect to a MongoDB instance requires authentication:
+
+- You can specify the username, password, and authentication database 
+  in the :manual:`connection string </reference/connection-string>`. 
+  For example, to connect to a remote MongoDB instance and
+  authenticate against the ``admin`` database as user ``alice``:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://alice:<password>@mongodb0.examples.com:28015/?authSource=admin"
+
+- You can use the ``--username``, ``--password``, and
+  ``--authenticationDatabase`` command-line options. For example, to
+  connect to a remote MongoDB instance and authenticate against the 
+  ``admin`` database as user ``alice``:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://mongodb0.example.com:28015" --username alice --password <password> --authenticationDatabase admin
+
+  .. note::
+
+     If you don't provide a ``--password`` value, ``mongosh`` prompts 
+     you for one.
+
+Connecting to a Replica Set
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a replica set:
+
+- You can specify the replica set name and members in the
+  :manual:`connection string </reference/connection-string>`:
+
+  .. code-block:: sh
+
+     mongosh "mongodb://mongodb0.example.com.local:27017,mongodb1.example.com.local:27017,mongodb2.example.com.local:27017/?replicaSet=replA"
+
+- If you are using the :manual:`DNS Seedlist Connection Format 
```

Comment:
I'm just wondering if the DNS Seedlist Format should come first. I don't know or feel strongly about this, but I'm wondering if because it's generally simpler we should be pushing users more toward it. Just something to think about.

Reply:
Makes sense to me, done.